config/agencies.yml

```diff
@@ -396,3 +396,9 @@
   name: Territory of Puerto Rico
 204:
   name: Territory of U.S. Virgin Islands
+205:
+  name: Administrative Office of the U.S. Courts
+206:
+  name: Defense Information Systems Agency
+207:
+  name: Legislative Branch
```

Comment:
is there a way we can pull this from identity-idp-config instead of having to update a second copy?